Formula/tree-sitter.rb

```diff
@@ -15,16 +17,26 @@ class TreeSitter < Formula
     sha256 "fbef61fe89cc07c5af05a06185ccb91cfa9c34773ab6b26b751ae0171e3806a2" => :mojave
   end
 
+  depends_on "emscripten" => [:build, :test]
+  depends_on "node" => [:build, :test]
   depends_on "rust" => :build
-  depends_on "node" => :test
 
   def install
+    cd "lib/binding_web" do
+      system "npm", "install", *Language::Node.local_npm_install_args
+    end
+    system "script/build-wasm"
+
```

Comment:
I also meant these needed to go at the end of the install block. This formula is for the tree-sitter library, so any additional steps should probably go after that.

Reply:
Anyway, it's fine. I'll just fix it after. Thanks!

Reply:
The install process is order sensitive. Because during building tree-sitter, [here](https://github.com/tree-sitter/tree-sitter/blob/1dc127e5dae7550a58a09886a19fdea19196bcf1/cli/build.rs#L19) use the wasm file to determine the build configuration.

Reply:
I see. That's fine then.